examples/jsm/loaders/lwo/IFFParser.js

```diff
@@ -676,8 +679,8 @@ class IFFParser {
 		this.currentPoints = [];
 		for ( var i = 0; i < length / 4; i += 3 ) {
 
-			// z -> -z to match three.js right handed coords
-			this.currentPoints.push( this.reader.getFloat32(), this.reader.getFloat32(), - this.reader.getFloat32() );
+			// x -> -x to match three.js right handed coords
+			this.currentPoints.push( - this.reader.getFloat32(), this.reader.getFloat32(), this.reader.getFloat32() );
```

Comment:
What is the coordinate system of LWO? I suspect it is left-handed but with what kind of up vector?

`three.js` is right-handed, `+Y up`.

Reply:
LWO: 
+Y = Up
+Z = Front
+X = Right

![image](https://github.com/mrdoob/three.js/assets/1384537/3face71a-4e16-4db6-ad46-e4fdf0f5f734)

Reply:
I've always thought that if you want to switch from left to right-handed (or vice versa) you just invert the basis's forward vector. Which is essentially flipping the sign of `z`. TBH, I'm a bit surprised that it's necessary in context of LWO/three.js to invert the right vector. 

I see you are an active LWO user and if you say that the default forward direction now matches between Lightwave and three.js (assuming you have applied not rotation to the object), I'm willing to accept the PR.

Reply:
I can send you the

Reply:
I can send you a diff of the amount of hacks I've been able to remove just to

- get geometry and pivot points to actually "just work" and
- the sheer simplification of having `lookAt()` "just work"

And as a side note, the simplified logic allowed me to attempt a different model structure which caused me to not rip my hair out. So now I didn't need to use circle intersection logic and can just use simple trig to get the desired model movements.

(I will have to post the code diff later, I am currently at softplay with my daughter)